ios/brave-ios/Sources/Onboarding/WelcomeFocus/FocusP3AScreenView.swift

```diff
@@ -129,6 +125,12 @@ struct FocusP3AScreenView: View {
           .padding(.vertical, 16)
           .padding(.horizontal, 20)
         }
+        .onAppear {
+          isP3AEnabled = p3aUtilities?.isP3AEnabled ?? true
```

Comment:
should this assignment happen in init? or `.onChange(of: initial:)`

Reply:
No you shouldn't assign `@State` var's on a View's init